File: Src/ButterflyNet.Parser/Scopes/SmallScope.cs
namespace ButterflyNet.Parser.Scopes {
	public class SmallScope : InlineScope {
		public override void Open(ButterflyAnalyzer analyzer) {
			analyzer.OpenSmallText();
		}

		public override void Close(ButterflyAnalyzer analyzer) {
			analyzer.CloseSmallText();
		}
	}
}

File: src/js/src/scopes/SmallScope.js
function SmallScope() {
	this.type = ScopeTypeCache.small;
	this.level = ScopeLevel.inline;
	this.canNestText = true;
	
	this.open = function(analyzer) {
		analyzer.openSmallText();
	};
	this.close = function(analyzer) {
		analyzer.closeSmallText();
	};
}

extend(Scope, SmallScope);